cbor.js

```diff
@@ -389,16 +523,16 @@ function decode(data, tagger, simpleValue) {
   }
 
   var ret = decodeItem();
-  if (offset !== data.byteLength)
-    throw "Remaining bytes";
+    if (offset !== data.byteLength && !allowRemainingBytes)
+      throw new Error((data.byteLength - offset) + " remaining bytes after end of encoding");
   return ret;
 }
 
 var obj = { encode: encode, decode: decode };
 
 if (typeof define === "function" && define.amd)
   define("cbor/cbor", obj);
-else if (typeof module !== "undefined" && module.exports)
+else /* istanbul ignore if */ if (typeof module !== "undefined" && module.exports)
```

Comment:
this is a feature check that can't get coverage testing; it needs to be excluded or needs to be removed